sal_interfaces/Dome/Dome_Commands.xml

```diff
@@ -13,16 +13,9 @@
     <Action></Action>
     <Value></Value>
     <Explanation>http://sal.lsst.org</Explanation>
-    <item>
-        <EFDB_Name>dirMotion</EFDB_Name>
-        <Description>Specifies de direction of crawl motion in azimuth: CW or CCW</Description>
-        <IDL_Type>string</IDL_Type>
-        <Units>unitless</Units>
-        <Count>1</Count>
-    </item>
     <item>
         <EFDB_Name>azRate</EFDB_Name>
-        <Description>Specifies the speed of crawl motion in degrees per second</Description>
+        <Description>Specifies the signed velocity of crawl motion in degrees per second. Positive means in the direction of increasing azimuth, negative in the direction of decreasing azimuth.</Description>

```

Comment:
I gently suggest removing "signed" here and below. The test of your helpful text makes that quite clear, and as Brian Stalder suggested on the Jira ticket, our convention is that velocity is signed and speed is not.

Reply:
Done